### DISCUSSION TITLE
Home page title

### DISCUSSION THREAD

**Discussion by maheshdharhari** [mmistakes/minimal-mistakes#5001] [General]
**Why is the title of the webpage being displayed as 'Blog - Blog'? Is there a specific reason why the same title is repeated and separated by a hyphen, or should we update the configuration to prevent this repetition?**

![image](https://github.com/user-attachments/assets/ef526c03-15bc-4efd-91c9-105ee5c66b57)

**Comment by whoami730** [comment]
I too have the same question. Were you able to find an answer for this?
